lighthouse-core/test/audits/viewport-test.js

```diff
@@ -38,4 +38,16 @@ describe('Mobile-friendly: viewport audit', () => {
       Viewport: 'width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1'
     }).rawValue, true);
   });
+
+  it('passes when a viewport with spaces is provided', () => {
```

Comment:
instead of separate tests, what if we expand the "'passes when a viewport is provided" test with a few permutations. Could just rename it to 'passes when a valid viewport is provided'. WDYT?